### ISSUE TITLE
Document how to overload rich mime types with Latexify 

### ISSUE THREAD

**Issue by ChrisRackauckas** [korsbo/Latexify.jl#94]
In Jupyter notebooks and such I would like to have Latexify be the output by default. Is there a nice way to do that? It would be nice to have the solution documented.

**Comment by korsbo** [comment]
I'm not sure what you mean. Do you essentially want `display(my_ode_network)` to hook in to latexify?

**Comment by korsbo** [comment]
If so, then I guess that something like 
```julia
Base.show(io::IO, ::MIME"text/tex", ode::DiffEqBase.AbstractParameterizedFunction) = Base.show(io, latexify(ode))
```
should work.

**Comment by ChrisRackauckas** [comment]
Yeah, that's good. I started thinking though... maybe if it's huge I need to set it to only print a bit of it? Is that possible?

**Comment by korsbo** [comment]
It's possible to leave out vector/matrix rows in the latexrecipe but this does not leave any `...` or something to indicate that the results have been truncated. It's a feature that I would like to add but it will have to wait until I have a little bit of free time.
